api/pom.xml

```diff
@@ -134,49 +74,67 @@
                 <targetPath>META-INF</targetPath>
             </resource>
         </resources>
-
         <plugins>
-           <plugin>
+           <!-- Sets minimal Maven version to 3.5.4 -->
```

Comment:
Well that's annoying.  I'm still on 3.3.x on most of my machines.

Reply:
Okay, I'll use 3.3.0 then. The reason I added it is that Maven produced warnings for not having a minimal version set, and I was addressing these warnings.